packages/components-css/card-group/index.scss

```diff
@@ -0,0 +1,28 @@
+.frameless-card-group {
+  container-type: inline-size;
+  display: grid;
+  gap: 24px;
```

Comment:
Kunnen we deze als design token toevoegen?

Dus `var(--frameless-card-group-gap)` en dan de `24px` in de JSON.